### PR TITLE
Landing page redesign

### PR DESCRIPTION
**Pull request by jcfr**
Implementation of the design available here: https://www.figma.com/file/Zi0KeHLpLecur4Q6ni0xQK/Slicer-Landing-Page?node-id=0%3A1

Preview is available at: https://deploy-preview-16--slicer-org.netlify.app/

### CODE REVIEW

**Comment by jcfr** [comment]
Slicer solution overlay issue:

image

**Comment by jcfr** [comment]
Fix the following links:
* Solutions
* Community
* Contact
* Resources
* About

**Comment by fedorov** [comment]
Would be great to see a functional preview of the design proposed in this PR!

**Comment by pieper** [comment]
@fedorov you can click on "Details" of the netlify check: https://deploy-preview-16--slicer-org.netlify.app/

**Comment by fedorov** [comment]
@pieper thanks, I missed it!

I like the top part, and the second panel, but then it gets really busy. I think the font needs to be spaced better, and maybe the amount of text needs to be reduced. I think the styling overall is not 

I also think it would be better if the page could be split into sections that could appear at the top of the page to facilitate navigation. Something like is done in this page: http://qiicr.org/dicom4miccai/ (see items in the top right that scroll to a position in the page).

I am not sure how to use Figma to communicate this feedback effectively. Hope this is ok.

**Comment by jcfr** [comment]
Thanks for the feedback.

## Issues

### Spacing

> I think the font needs to be spaced better

Agreed, these screenshots may be helpful illustrating:

* figma:
image

* website:
image

> and maybe the amount of text needs to be reduced.

@fedorov Could you provide suggestions here: https://docs.google.com/document/d/1MFHUKaY1K7ZjlMUT6O-L1tCj2yIhhdtzimBbL5lJrfc/edit#heading=h.jh7so2nj3dc

## Section at the top

This is now tracked in a dedicated issue: https://github.com/Slicer/slicer.org/issues/17

For now, we will move forward with the current proposal and revisit this at a later time.

# Sharing Feedback

Generally speaking, you should be able to add comment in figma:
image

That said, since some of the issue are specific to the implementation, it makes sense to communicate these here.

# Timeline

Considering the Kitware design team will shift focus next week, we will stick with the design described in [figma](https://www.figma.com/file/Zi0KeHLpLecur4Q6ni0xQK/Slicer-Landing-Page?node-id=0%3A1).

**Comment by lassoan** [comment]
I think renaming "3D Slicer Functionalities" to "Features" would be better. Also, I find the "3D Slicer" word to be too repetitive - we should have it in some heading pinned to the top and remove the "3D Slicer" prefix from these headings: Functionalities, Extensions, Solutions, Commercial Use, Community.

**Comment by jcfr** [comment]
> re: button grid:

I will also implement the following:

"Download Slicer" -> "Download"
"Discussion Forum" -> "Forum"
"Developer Resources" -> "Developer"

| Before | After |
|----|----|
|image | image |

File: _includes/features.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
       </div>
       <div class="container is-flex" style="flex-wrap: wrap;">
         {% for item in page.features_list %}
-        <div class="features">
+        <div class="feature">
```
Comment (jcfr) [inline _includes/features.html:9]: Thanks. Good catch

**Comment by lassoan** [comment]
Button grid changes look very good. We should probably also change:
- "Manual" -> "Documentation" (it is not completely clear what "Manual" means; "User Manual" would be clear but it would be a bit too restrictive)
- "Developer" -> "Developers" (I've checked a few websites and developer section is usually called "Developers")

**Comment by fedorov** [comment]
@jcfr did you experiment with laying out the tiles in a single row/column below the logo and Slicer blurb?

**Comment by jcfr** [comment]
> laying out the tiles in a single row/column below the logo and Slicer blurb?

Something like this ?

image

**Comment by fedorov** [comment]
No, I meant something like this:

image

**Comment by jcfr** [comment]
@swederik Here is the work in progress new landing page for Slicer, would be great to get your feedback.

See https://deploy-preview-16--slicer-org.netlify.app/

cc: @lassoan @pieper

**Comment by lassoan** [comment]
I think this is almost ready to go! I have just a few suggestions (highest priority first):

- [ ] Slicer in action section content: This must be fixed. We could add links to youtube videos and discourse posts here. Can we send a pull requests to add real content?
- [ ] Fix style of "Extensions" and "Community" sections. The first 3 sections are alternating shades of grey (top, What is 3D Slicer, Features) - it is good; but then "Extensions" section uses a completely different style (it appears as "disabled", does not have drop shadow as the first 3 sections); then Solutions and Commercial Use sections are good and consistent; then "Community" is again an outlier (completely different style, smaller window, with text background, does not provide link when hovering over it).
- [ ] Update links to use read-the-docs instead of old wiki in the bottom section (User Q&A, Feedback, Developer FAQ, Contact, Licencing, Citing, Commercial use, Acknowledgements, Community)
- [ ] Make "What is 3D Slicer" carousel move faster (I scroll through in 2-3 seconds, in that time the carousel does not move, so I don't notice that there is more content there)
- [ ] Add links to descriptions in features section
- [ ] Clean up the top menu (that appears when you scroll down the page a bit) and footer (Application, Support, Solutions, ...): since we already have all the commonly needed links in the top section and other useful links in the footer, this section with some selected links is quite redundant and confusing. I think we can keep just a "back to top" menu item, or hide it completely. We could use it as a sitemap in the future, but since sections are not very well balanced and some of them are very short, it would probably not be that helpful.

**Comment by jamesobutler** [comment]
> Make "What is 3D Slicer" carousel move faster (I scroll through in 2-3 seconds, in that time the carousel does not move, so I don't notice that there is more content there)

Appears that this is likely controlled by the following
https://github.com/Slicer/slicer.org/blob/f1fa59b8c3e157c2b648994fbddc68ed377d80d5/_includes/about.html#L30

**Comment by lassoan** [comment]
@jamesobutler thanks a lot for looking into these. I did not have time to figure out how to get the background right, but fixed the carousel content and speed and a lot of links.

See update result here:
https://deploy-preview-16--slicer-org.netlify.app/

**Comment by jamesobutler** [comment]
I issued PR #70 targeting this branch with some logo changes

**Comment by jcfr** [comment]
> Clean up the top menu

All set. Navbar at the top is now consistent with the footer. And content available in both `_data/footer-site-map.yml` and `_data/solutions.yml` is also used to generate it.

image

**Comment by lassoan** [comment]
The preview on netlify looks great! I think this is ready to go.

Should we send out the link for final blessing to the core team?
Should we post a message on discourse to give the community a chance to give feedback?

**Comment by jcfr** [comment]
> Should we send out the link for final blessing to the core team?
> Should we post a message on discourse to give the community a chance to give feedback?

Both, after I get final feedback from our UI team, I will do so.

**Comment by jamesobutler** [comment]
- [ ] On mobile when I scroll down the header that begins to show is extremely large and can only read like 1/3 of the page.

8847C38D-C427-4489-AC9B-6EA4E2498E4B

**Comment by lassoan** [comment]
I don't see this large header on mobile (Edge and Chrome browsers on Android). What is a little bit annoying is that the arrows in the carousel cover the text:

image

**Comment by jamesobutler** [comment]
My mobile experience was on iOS 14.4.1 using Safari on an iPhone XR.

**Comment by fedorov** [comment]
Looks nice! Just a couple of observations:

* I noticed "Spatial registration" panel is blinking on every carousel move:

* I think it would look better if there were 6, not 5 buttons to the right of the logo (red arrow below). How about adding a button for YouTube channel/tutorial videos?

**Comment by jamesobutler** [comment]
I'm observing that blinking of "Spatial registration" whenever my mouse hovers over the Next buttons for the carousel. Carousel doesn't have to necessarily be moving.

**Comment by lassoan** [comment]
> I think it would look better if there were 6, not 5 buttons to the right of the logo (red arrow below). How about adding a button for YouTube channel/tutorial videos?

We don't have a youtube channel (each group tend to upload videos on its own channel), but maybe we could create playlists.
Until then, we could add a twitter button there.

**Comment by fedorov** [comment]
> We don't have a youtube channel (each group tend to upload videos on its own channel), but maybe we could create playlists. Until then, we could add a twitter button there.

I thought we do, and I thought we were adding videos from external channels to curated playlists here: https://www.youtube.com/channel/UC11x1iQ7ydSIFYw4L6wveXg ?

Anyway, twitter is fine too.

**Comment by pieper** [comment]
> I thought we do, and I thought we were adding videos from external channels to curated playlists here: https://www.youtube.com/channel/UC11x1iQ7ydSIFYw4L6wveXg ?

That has 3k members and almost 300k views, but as I recall we don't know who owns the channel.

**Comment by fedorov** [comment]
Well, maybe we don't know who owns it now, but I had the password, and passed it along to @jcfr last Summer.

**Comment by jcfr** [comment]
> That has 3k members and almost 300k views, but as I recall we don't know who owns the channel.

I know have access .. and need to perform the relevant bookkeeping, in the meantime I will had the twitter button

**Comment by pieper** [comment]
> > That has 3k members and almost 300k views, but as I recall we don't know who owns the channel.
> 
> I now have access .. and need to perform the relevant bookkeeping, in the meantime I will had the twitter button

I'd vote for youtube then, since a lot of people learn best through that medium.

**Comment by lassoan** [comment]
Yes, we have access to the channel but it lacks content and I don't see how we could keep it up-to-date. We only have 3 original videos there and the Slicer core team would be only be able to produce a small amount (we could upload feature demos and tutorials there). We could maintain playlists, but it would require quite a lot of work to be relevant and useful. The channel has 3k subscribers, which is quite significant, so it would be good to revive it.

I like how twitter works - that anybody can bring a post to the attention of the core team (by mentioning @3dslicerapp) and it can be liked/retweeted with a single click.

**Comment by jamesobutler** [comment]
I think would make sense to revamp the YouTube as part of revamping of the "Tutorials" as the YouTube videos appear to be primarily tutorials instead of showcases. The tutorials, whether in written format or video, often struggle with maintenance as the GUI and other functionality changes to where it is only ever "ok/good" and not "great".

**Comment by fedorov** [comment]
I just noticed button colors match those of the logo squares - very nice!

> Yes, we have access to the channel but it lacks content and I don't see how we could keep it up-to-date. 

Adding videos to playlists is as easy as bookmarking. 

Personal preferences between Twitter and YouTube aside - we have 2x as many subscribers on that YouTube channel as followers on Twitter.

**Comment by lassoan** [comment]
We don't need to choose between Youtube and Twitter. We can advertise both.

Twitter is active, all contents are relevant, we see lots of very useful feedbacks, interactions there. So, it is ready to be advertised on the frontpage.

Youtube is essential - if something is not on youtube it does not exist. So, when somebody finds the time to clean up the youtube channel add new content (and de-emphasize old content) then we can start advertising it. Maybe we should ask on the Slicer forum if a few people would volunteer for this work? (we could ask in the same post where we ask community for feedback on the website)

**Comment by jamesobutler** [comment]
As it relates to the main buttons on the homepage:
- YouTube should be linked from the page that the "Training" button goes to (https://www.slicer.org/wiki/Documentation/4.10/Training). This is because the purpose of the content is the same. For Tutorials/Training.  Currently YouTube doesn't appear to be used for anything else like video announcements or anything.
- The "Twitter" button is fine as it provides something different. It provides Social content which is something new in that set of buttons.

**Comment by jcfr** [comment]
> I think would make sense to revamp the YouTube as part of revamping of the "Training" (aka Tutorials) as the YouTube videos appear to be primarily tutorials instead of showcases. The tutorials, whether in written format or video, often struggle with maintenance as the GUI and other functionality changes to where it is only ever "ok/good" and not "great".

@jamesobutler 

To follow up on this, we were able to figure out the details and have now shared ownership of the youtube channel. Let us know if you would like to have a more active role, and we could also grant you access. 

In the mean time, relevant link have been added to  the site. See https://github.com/Slicer/slicer.org/pull/16/commits/c23f089ce7577c8bc2946613d34fea8efeed7909

Corresponding channel is https://www.youtube.com/c/3DSlicerChannel

**Comment by fedorov** [comment]
I just realized that the page does not even try to acknowledge support of funding agencies! NIH is not even mentioned on the page! This is not good at all. I would most definitely acknowledge funders (e.g., "Development was made possible through research grants from NIH (NCI, NHLBI, ...), ...", and maybe even acknowledge main grant numbers that supported the development, and definitely include a link to the wiki page with the full list of current and past grants: https://www.slicer.org/wiki/Documentation/4.x/Acknowledgments!

image

**Comment by jcfr** [comment]
> I just realized that the page does not even try to acknowledge support of funding agencies!

This done here:

image


... and then we have the "Funding source" section on the acknowledgments page:

image


Source: https://slicer.readthedocs.io/en/latest/user_guide/about.html#funding-sources

**Comment by lassoan** [comment]
I agree with @fedorov, considering the amount of funding Slicer received from NIH, we should mention it on the frontpage. I added two sentences here: https://deploy-preview-16--slicer-org.netlify.app/#features

**Comment by jamesobutler** [comment]
The color of the 6 main buttons match the 6 six main colors from the logo.  There is also 5 matching colors at the top of the header. We should have the 6th color up there as well for consistency.
image

**Comment by lassoan** [comment]
Speaking of button colors... These colors look nice and vibrant in the logo because they are small, but for me this block of buttons is just too large for these harsh error-like colors. The blue and green buttons are nice to look at but the reds and oranges are a bit too strong for me, they are a bit alarming, gives the impression that something is wrong. Maybe we could keep the hue of these colors but tone all of them down (reduce their saturation). It would also reduce cognitive dissonance for me if the twitter button got the blue color.

**Comment by jcfr** [comment]
After addressing https://github.com/Slicer/slicer.org/issues/76, I will move forward.

This will  likely happen after I am back from the :climbing: gym

**Comment by jcfr** [comment]
Closing. This topic associated with this PR has been pushed as https://github.com/Slicer/slicer.org/tree/master

**Comment by lassoan** [comment]
Have we asked feedback on the new page on the Slicer forum? Do we want to ask now?

**Comment by jcfr** [comment]
> Have we asked feedback on the new page on the Slicer forum? Do we want to ask now?

Will do so now, having the workaround for the commercial use, it makes sense to more broadly advertise it.

**Comment by jcfr** [comment]
For reference, see https://discourse.slicer.org/t/its-all-about-transitions-lets-talk-about-slicers-landing-page/113/44?u=jcfr

**Pull request closed without merging**